runtimes/peregrine/Cargo.toml

```diff
@@ -68,6 +68,7 @@ pallet-transaction-payment = {git = "https://github.com/paritytech/substrate", d
 pallet-treasury = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.13"}
 pallet-utility = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.13"}
 pallet-vesting = {git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.13"}
+pallet-proxy = {branch = "polkadot-v0.9.13", default-features = false, git = "https://github.com/paritytech/substrate"}
```

Comment:
We try to keep this list sorted, so I would put it just after `pallet-randomness-collective-flip` :)

Reply:
Perhaps also re-arranging `branch`, `default-features` and `git` to match all the others...

Reply:
done